File: src/main/java/com/ocunha/configuration/ApplicationConfiguration.java
package com.ocunha.configuration;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.Configuration;

@Configuration
public class ApplicationConfiguration {

    @Value("${api.result.size:5}")
    private Integer apiResultSize;

    public Integer getApiResultSize() {
        return apiResultSize;
    }
}

File: src/main/kotlin/com/ocunha/configuration/ApplicationConfiguration.kt
package com.ocunha.configuration

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.Configuration

@Configuration
class ApplicationConfiguration {

    @Value("\${api.result.size:5}")
    lateinit var apiResultSize: String

    fun getApiResultSize(): Int {
        return apiResultSize.toInt()
    }
}